ui/lib/pki/addon/templates/certificates/index.hbs

```diff
@@ -1,12 +1,23 @@
+<SecretListHeader
+  @model={{this.model.parentModel}}
+  @backendCrumb={{hash
+    label=this.model.parentModel.id
+    text=this.model.parentModel.id
+    path="vault.cluster.secrets.backend.list-root"
+    model=this.model.parentModel.id
+  }}
+  @isEngine={{true}}
+/>
+{{outlet}}
 <Toolbar>
-  {{#if (gt this.model.length 0)}}
+  {{#if (gt this.model.certificateModel.length 0)}}
```

Comment:
If this is always an array you could just look for length as truthy:
`{{if this.model.certificateModel.length}}`

There are some instances I see below using `gt` if you decide to change it.

Reply:
Good idea! Thanks.